myjquery.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+	
+        // console.log('I got here!');
+
+
+$(".readmore").click(function(){
```

Comment:
All of these event handlers need to call `event.preventDefault()`. Currently, when you click on any of yours links, the scroll bar jumps to the top of the page. This is how to prevent it:

https://ga-students.github.io/fewd-dc-31/08_Intro_jQuery/#/18/3